utils/test_cart_lib.sh

```diff
@@ -57,7 +57,9 @@ else
     echo "checking libcart.so"
     nm -g "${SL_PREFIX}/lib/libcart.so" |
         grep -v " U " |  grep -v " w " |  grep -v " crt_" | grep -v " swim_" |
-        grep -v "D CMF_" | grep -v "D CQF_" |
+        grep -v "D CMF_" | grep -v "D CQF_" | grep -v "d_fault_attr_mem" |
+        grep -v "d_fault_id_mem" |
+        grep -v "\bd_\w*_logfac\b" |
```

Comment:
I thought I removed d_log_fac.   Are you seeing it again?  If so, it is a regression

Reply:
it doesn't report error when I remove this. will remove.